extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -206,6 +208,11 @@ export class BookTocManager implements IBookTocManager {
 		}
 		if (isModified) {
 			await fs.writeFile(tocPath, yaml.safeDump(this.tableofContents, { lineWidth: Infinity, noRefs: true, skipInvalid: true }));
+			let toc: undoRedoToc = {
```

Comment:
nit - but `toc` isn't a great name here. This isn't a toc - it's more a state object for undoing/redoing (see comment about class name I gave)